SPECS/annobin/annobin.spec

```diff
@@ -34,29 +21,189 @@ URL:     https://fedoraproject.org/wiki/Toolchain/Watermark
 # Use "--with debuginfod" to force support for debuginfod to be compiled into
 # the annocheck program.  By default the configure script will check for
 # availablilty at build time, but this might not match the run time situation.
+# FIXME: Add a --without debuginfod option to forcefully disable the configure
+# time check for debuginfod support.
 %bcond_with debuginfod
 
+# Use "--without clangplugin" to disable the building of the annobin plugin for Clang.
+%bcond_without clangplugin
+
+# Use "--without gccplugin" to disable the building of the annobin plugin for GCC.
+%bcond_without gccplugin
```

Comment:
gccplugin flag will always be true as we do not pass bcond_without flag for anything.  Do we want to use these bcond_without then?